FV3/gfsphysics/GFS_layer/GFS_physics_driver.F90

```diff
@@ -4557,6 +4557,8 @@ subroutine GFS_physics_driver                                 &
               Diag%zhao_carr_physics%cloud_water = (Stateout%gq0(1:im,1:levs,ntcw) - dqdt(:,:,ntcw)) / dtp
               Diag%zhao_carr_physics%temperature = (Stateout%gt0(1:im,1:levs) - dtdt) / dtp
             end if
+            ! rain1 has units m, and represents surface precip over dtp
+            Diag%zhao_carr_physics%surface_precipitation = rain1 / dtp * 1000.0
```

Comment:
Here and below you could consider using `rhowater` from the `physcons` module to make the units a little clearer:
https://github.com/ai2cm/fv3gfs-fortran/blob/7b6481dfbfa3956c9e8913e75ac03f4f31b8243d/FV3/gfsphysics/physics/physcons.F90#L176

Reply:
good idea. i didn't know about this constant.